distributed/tests/test_scheduler.py

```diff
@@ -1234,6 +1255,42 @@ async def test_profile_metadata_keys(c, s, a, b):
     )
 
 
+@gen_cluster(
+    client=True,
+    config={
+        "distributed.worker.profile.interval": "1ms",
+        "distributed.worker.profile.cycle": "100ms",
+    },
+)
+async def test_statistical_profiling(c, s, a, b):
+    futures = c.map(slowinc, range(10), delay=0.1)
+
+    await wait(futures)
+
+    profile = await s.get_profile()
+    assert profile["count"]
```

Comment:
Why this test?  Didn't we already test this functionality?

Reply:
Looks like we don't have any explicit unit test for `Scheduler.get_profile` but rather a bunch of `Client.profile` tests which essentially just call into the handler.
I can understand both perspectives since the client is user facing API but otoh the dashboard calls Scheduler.get_profile directly.